### DISCUSSION TITLE
How to reset selected date

### DISCUSSION THREAD

**Discussion by cjayyy** [easepick/easepick#22] [Q&A] [answered]
When we need to reset the date (or date range) programmatically - how can that be achieved? 
I was looking for some reset method but didn't find one. So I tried to use `setDateRange()` but that doesn't seem to accept any argument that would do a reset. Am I missing something here?

**Comment by wakirin** [comment, marked as answer]
The new method (`clear`) will be added in version 1.1.0
New release in development.

**Reply by cjayyy** [reply]
Great! Thanks for the info :)

**Reply by wakirin** [reply]
Method `clear` added. 
Use 1.1.3 version.

https://jsfiddle.net/waki/s4twb23z/

**Reply by cjayyy** [reply]
Thank you

**Reply by cjayyy** [reply]
Btw. haven't you forgot to add type definitons? Or is fail on my side? 

`Property 'clear' does not exist on type 'Core'.ts`

**Reply by wakirin** [reply]
Make sure you are using version 1.1.3.
If you are using `@easepick/bundle` you will need to reinstall it to update the dependencies.
```
npm remove @easepick/bundle
npm install @easepick/bundle
```
or 
remove `package-lock.json` and then `npm install`

You can see that the `clear` method exists in `.d.ts` file: https://cdn.jsdelivr.net/npm/@easepick/core@1.1.3/dist/core.d.ts

**Reply by cjayyy** [reply]
You are right, I needed to reinstall the dependency, the update wasn't enough. Thx
